pyacq/devices/tests/test_eeg_emotiv.py

```diff
@@ -0,0 +1,81 @@
+import time
+import pytest
+
+from pyacq.devices.eeg_emotiv import Emotiv, HAVE_PYCRYPTO  # , HAVE_PYWINUSB
+from pyacq.viewers.qoscilloscope import QOscilloscope
+from pyqtgraph.Qt import QtCore, QtGui
+
+from collections import OrderedDict
+import os
+
+
+# Manual scan of devices
+def get_available_devices():
```

Comment:
I want get_available_devices() that give a device list that also contain possible params for Node.configure().
For that Node anf PyAudio it is easy.
For measurement computing, you can have 2 subdevice (analog and digital) and parameters for each channel.
I would like the get_available_devices() to give default parameters (maybe in pg.parameteres.tree format).
This could hekp for generating UI for devices parameterization.